### PR TITLE
Use fetch-translations script from helsinki-utils

### PR DESCRIPTION
**Pull request by Jokinen**
Replaces copy pasted script with imported one from `helsinki-utils`.

### CODE REVIEW

**Review by tuovinensanttu — APPROVED** [review]
Looks good 👍 I think it's a nice idea that you used another approach here. Now there is two working examples how to use this new utility.

File: package.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "ci": "CI=true yarn test --coverage",
     "lint": "eslint --ext js,ts,tsx src",
     "codegen": "apollo client:codegen ./src/graphql/generatedTypes.ts --outputFlat --includes=./src/**/*.graphql --target=typescript --endpoint=https://profiili-api.test.kuva.hel.ninja/graphql/ --useReadOnlyTypes --addTypename",
-    "update-translations": "ts-node  -P ./scripts/tsconfig.json -r dotenv/config scripts/update-translations.ts"
+    "update-translations": "ts-node -P ./scripts/tsconfig.json -r dotenv/config --files scripts/update-translations.ts"
```
Comment (badrange) [inline package.json:56]: Didn't know about this way to read config, nice!

File: scripts/helsinki-utils.d.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+declare module 'helsinki-utils/scripts/fetch-translations' {
```
Comment (badrange) [inline scripts/helsinki-utils.d.ts:1]: Did you consider adding the types in a PR towards helsinki-utils?

**Review by badrange — APPROVED** [review]
This solution is fine! 
If you remove the local script that calls the one from the library you might get away from having ts-node as a dependency of this project (unless there is a need from somewhere else..)

File: scripts/helsinki-utils.d.ts (hunk 2 above)
Comment (Jokinen) [reply scripts/helsinki-utils.d.ts:1]: Yup 👍I wrote down an issue to track it: https://github.com/City-of-Helsinki/helsinki-utils-js/issues/5

**Comment by Jokinen** [comment]
By doing something magical, I've managed to install the package so that it has been recorded within the lock file, but not within package.json itself. I'll fix this before merge.

**Comment by Jokinen** [comment]
@badrange The original script pulled some configs from the ENV. I wanted to avoid removing that functionality, because I was not sure how it was used. I felt that this was the quickest way to retain that functionality.

But you are definitely correct. If the configuration is not used, simplifying the script would remove complexity in configuring, dependencies and code.

**Comment by Jokinen** [comment]
I had also forgotten to remove those dependencies that are now depended on by the `helsinki-utils` repo. `node-fetch` (+types) and `i18n-csv-to-json-converter`. I've removed these.

**Pull request merged**